bin/runnable-upload.js

```diff
@@ -6,13 +6,16 @@ var runnable = require('../lib/runnable')
 
 program
   .arguments('<file>')
+  .arguments('<file>', 'Destination path in then container for the specified file', '')
```

Comment:
close. what you want is just `.arguments('<file> [dest]')`. then the code below will still work as well.